topics/protocol.md

```diff
@@ -306,7 +306,7 @@ The client sends the command **LLEN mylist** in order to get the length of the l
 
     S: :48293\r\n
 
-As usual we separate different parts of the protocol with newlines for simplicity, but the actual interaction is the client sending `*2\r\n$4\r\nLLEN\r\n$6\r\nmylist\r\n` as a whole.
+As usual we separate different parts of the protocol with newlines for simplicity, but the actual interaction is the client sending "*2\r\n$4\r\nLLEN\r\n$6\r\nmylist\r\n" as a whole.
```

Comment:
Suggestion: Use backticks inside quotes in this and many other RESP strings for even better readability.

```suggestion
As usual we separate different parts of the protocol with newlines for simplicity, but the actual interaction is the client sending "`*2\r\n$4\r\nLLEN\r\n$6\r\nmylist\r\n`" as a whole.
```